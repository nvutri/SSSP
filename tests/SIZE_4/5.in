c random graph n=32 m=128 lo=0 hi=1024 seed=1053106347
p sp 32 128
a 21 29 884
a 29 1 567
a 1 30 562
a 30 14 359
a 14 22 963
a 22 18 914
a 18 9 224
a 9 17 666
a 17 3 384
a 3 19 740
a 19 8 741
a 8 20 204
a 20 28 250
a 28 5 466
a 5 12 947
a 12 25 48
a 25 32 481
a 32 13 22
a 13 24 49
a 24 6 48
a 6 15 814
a 15 31 718
a 31 27 195
a 27 7 520
a 7 10 391
a 10 26 877
a 26 11 33
a 11 4 425
a 4 16 406
a 16 2 529
a 2 23 1020
a 23 21 57
a 3 24 326
a 17 22 507
a 27 9 27
a 29 24 614
a 1 4 865
a 12 25 356
a 28 4 140
a 5 31 859
a 11 15 1004
a 32 3 772
a 29 21 780
a 20 17 154
a 1 4 40
a 17 15 435
a 30 29 732
a 9 7 86
a 1 14 105
a 17 19 916
a 10 20 265
a 7 16 81
a 29 10 335
a 14 8 255
a 9 11 8
a 30 1 646
a 13 15 460
a 19 11 24
a 7 11 1008
a 16 22 363
a 10 27 639
a 14 3 783
a 32 28 441
a 9 16 494
a 23 25 473
a 4 16 589
a 20 6 750
a 6 10 509
a 12 28 194
a 13 20 341
a 16 5 730
a 1 23 572
a 14 29 341
a 27 31 315
a 21 26 817
a 3 5 914
a 22 4 49
a 2 25 714
a 19 29 620
a 17 25 474
a 16 29 290
a 10 23 313
a 12 10 390
a 16 19 434
a 2 24 516
a 8 20 291
a 10 27 530
a 13 9 202
a 17 30 308
a 21 14 216
a 8 16 880
a 19 11 62
a 18 5 1020
a 25 20 731
a 4 19 716
a 5 6 395
a 30 2 723
a 27 4 241
a 11 26 798
a 2 25 171
a 4 10 484
a 2 28 246
a 24 9 907
a 32 6 719
a 20 17 146
a 13 1 129
a 24 5 192
a 27 26 439
a 12 7 45
a 21 12 545
a 2 17 599
a 12 21 35
a 12 31 407
a 25 8 358
a 20 6 40
a 1 29 153
a 3 6 427
a 4 24 922
a 4 32 436
a 3 19 179
a 26 11 86
a 9 26 62
a 30 12 406
a 14 3 306
a 25 10 679
a 24 26 931
a 17 1 704
a 4 14 514
