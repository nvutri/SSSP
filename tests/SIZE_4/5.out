1 0
2 1158
3 411
4 40
5 739
6 595
7 786
8 360
9 700
10 488
11 652
12 968
13 498
14 105
15 958
16 629
17 797
18 1906
19 590
20 651
21 629
22 992
23 572
24 547
25 1045
26 762
27 1018
28 901
29 153
30 562
31 1333
32 476
